code/modules/mob/living/say.dm

```diff
@@ -329,7 +329,12 @@ GLOBAL_LIST_INIT(department_radio_keys, list(
 	return 1
 
 /mob/living/proc/can_speak_vocal(message) //Check AFTER handling of xeno and ling channels
-	if(HAS_TRAIT(src, TRAIT_MUTE) && get_message_language(message) != /datum/language/signlanguage)
+	var/obj/item/bodypart/leftarm = get_bodypart(BODY_ZONE_L_ARM)
+	var/obj/item/bodypart/rightarm = get_bodypart(BODY_ZONE_R_ARM)
+	if(HAS_TRAIT(src, TRAIT_MUTE) && get_selected_language() != /datum/language/signlanguage)
+		return 0
+			
+	if (rightarm.is_disabled() && leftarm.is_disabled() && get_selected_language() == /datum/language/signlanguage)
```

Comment:
Is this intentionally written that _both_ arms are required to be disabled to be unable to use sign language?

Reply:
Yes- because if only one arm is broken, presumably you could still sign with the other. Though I get what you're getting at- perhaps I should make it appear "slurred" if one arm is broken.

Reply:
this is good enough for now then